provisioning/templates-api/common.sh

```diff
@@ -35,6 +35,7 @@ helm upgrade \
   --version "$ETCD_HELM_CHART_VERSION" \
   --values ./kubernetes/deploy/etcd/values.yaml \
   --namespace templates-api \
+  --set "replicaCount=$TEMPLATES_API_ETCD_REPLICAS" \
```

Comment:
Namiesto nahradzovania ENV v `values.yaml`, nastavime tuto hodnotu priamo cez HELM.
Vramci cron job-u pre defragmentaciu mi to robilo problemy, ked som potreboval pouzit znak `$`.